README.md

```diff
@@ -1,5 +1,9 @@
 # CoW AMM
 
+> [!WARNING]
+> All CoW AMM versions described in this repository have been deprecated in favor of the [Balancer CoW AMM](https://github.com/defi-wonderland/balancer-v1-amm/).
+> Pools deployed from the code in this repository are not going be supported in the future.
```

Comment:
Is there any timeline associated with this? If not, we should be explicit that support may drop without any subsequent notice.